xbmc/addons/interfaces/GUI/Window.cpp

```diff
@@ -316,11 +625,50 @@ void Interface_GUIWindow::add_list_item(void* kodiBase, void* handle, void* item
   }
 
   Interface_GUIGeneral::lock();
-  pAddonWindow->AddItem(pItem, item_position);
+  pAddonWindow->AddItem(pItem, list_position);
```

Comment:
The use as "list_position" is to match the for "C" standardized string format. The other is normally frowned upon for "C".